src/main.cpp

```diff
@@ -228,7 +228,7 @@ static int parse_argv(int argc, char *argv[], Configuration &config)
             break;
         }
         case 'l': {
-            config.log_config.logs_dir = strdup(optarg);
+            config.log_config.logs_dir.assign((const char *)optarg);
```

Comment:
Why is this better than `config.log_config.logs_dir = std::string(optarg)`? (Just out of curiosity.)

Reply:
OK. I only found the `=` assignment while skipping through the code, but I seems to have used both variants. So I might need to change it in the future to one consistent style.
As far as I have read, both variants should be the same, because `assign()` and the std::string c-tor both copy the C string contents. I just was curious, if I have missed something.